src/modules/approval-requests/translations/en-us.yml

```
@@ -0,0 +1,208 @@
+      screenshot: "https://drive.google.com/file/d/1gacesWd8MkyIY6UXvQSatv2ok67ZhbKk/view?usp=drive_link"
+      value: "Additional note"
+  - translation:
+      key: "approval-requests.request.approver-actions.submit-approval"
+      title: "Label for the submit approval button when approving an approval request"
+      screenshot: "https://drive.google.com/file/d/12C2F8zoCD_nMU-jSeu4kaMWwTeJNE3HJ/view?usp=drive_link"
+      value: "Submit approval"
+  - translation:
+      key: "approval-requests.request.approver-actions.submit-denial"
+      title: "Label for the submit denial button when denying an approval request"
+      screenshot: "https://drive.google.com/file/d/153Idb3CgJ_eFfWxLE4esAdB0DceOH8gj/view?usp=drive_link"
+      value: "Submit denial"
+  - translation:
+      key: "approval-requests.request.approver-actions.cancel"
+      title: "Label for the cancel button when approving or denying an approval request"
+      screenshot: "https://drive.google.com/file/d/1TjjF7CrDr-VYbZ-rpGs0pLCersevq9uL/view?usp=drive_link"
+      value: "Cancel"
+  - translation:
+      key: "approval-requests.request.notification.denial-submitted"
+      title: "Notification message shown when the denial of an approval request was submitted"
+      screenshot: "https://drive.google.com/file/d/1_Af2Xi2l6kcKI9QmFFQxH3SvI_Eaarg5/view?usp=drive_link"
+      value: "Denial submitted"
+  - translation:
+      key: "approval-requests.request.notification.approval-submitted"
+      title: "Notification message shown when the approval of an approval request was submitted"
+      screenshot: "https://drive.google.com/file/d/16BOEFXrbM29me3KdGgROerXXO5ns-7oh/view?usp=drive_link"
+      value: "Approval submitted"
+  - translation:
+      key: "approval-requests.list.header"
+      title: "Header for the approval requests list page"
+      screenshot: "https://drive.google.com/file/d/1MUj23YI3kUT-Udmh1LA0rt85-q078SMj/view?usp=drive_link"
+      value: "Approval requests"
+  - translation:
+      key: "approval-requests.list.search-placeholder"
+      title: "Placeholder for the search dropdown on the approval requests list page"
+      screenshot: "https://drive.google.com/file/d/10Mdnw0YkIq3w4CjdDOSD9efAAxDapzYr/view?usp=drive_link"
+      value: "Search approval requests"
+  - translation:
+      key: "approval-requests.list.no-requests"
+      title: "Text to convey that no approval requests exist on the approval requests list page."
+      screenshot: "https://drive.google.com/file/d/1rrZ1MZKAsY7mNo89McYaZNwEB5pDjKtk/view?usp=sharing"
+      value: "No approval requests found."
+  - translation:
+      key: "approval-requests.list.status-dropdown.label"
+      title: "Label for the status dropdown on the approval requests list page"
+      screenshot: "https://drive.google.com/file/d/18_4fS0yROAMC1yVdDYLfYvjKWeUdWCDH/view?usp=drive_link"
+      value: "Status:"
```

```suggestion
      value: "Status"
```
I think I missed this when I first reviewed those strings, but we never use colons for dropdown labels

---

Thanks for the tag and callout, @kubraokcu I can put up a PR to fix it on master today

---

@kubraokcu updated the translation in https://github.com/zendesk/copenhagen_theme/pull/589

---

Approving since the translation has been updated in #589 🙏🏼